pages/api/additives/index.js

```diff
@@ -0,0 +1,37 @@
+import dbConnect from "@/db/connect";
+import Additive from "@/db/models/Additive";
+import { getToken } from "next-auth/jwt";
+
+const secret = process.env.NEXTAUTH_SECRET;
+
+export default async function handler(request, response) {
+  await dbConnect();
+  const token = await getToken({ req: request, secret });
+
+  if (request.method === "GET") {
+    const additives = await Additive.find({ userId: token.sub }).sort("name");
+
+    return response.status(200).json(additives);
+  }
+  if (request.method === "POST") {
+    try {
+      let additiveData = request.body;
+      additiveData.userId = token.sub;
+
+      console.log("request", JSON.stringify(additiveData));
```

Comment:
You should generally not redeclare function parameters.

```javascript
const additiveData = {...request.body, userId: token.sub}

```

Reply:
done